PurchasesTests/Purchasing/OfferingsManagerTests.swift

```diff
@@ -0,0 +1,165 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  OfferingsManagerTests.swift
+//
+//  Created by Juanpe Catalán on 9/8/21.
+
+import XCTest
+import Nimble
+import StoreKit
+import PurchasesCoreSwift
+
+// TODO (post-migration): Move this test case to the new framework target (PurchasesCoreSwift).
+class OfferingsManagerTests: XCTestCase {
+
+    let mockDeviceCache = MockDeviceCache()
+    let mockOperationDispatcher = MockOperationDispatcher()
+    let mockSystemInfo = try! MockSystemInfo(platformFlavor: "iOS",
+                                             platformFlavorVersion: "3.2.1",
+                                             finishTransactions: true)
+    let mockBackend = MockBackend()
+    let mockOfferingsFactory = MockOfferingsFactory()
+    let mockProductsManager = MockProductsManager()
+    var offeringsManager: OfferingsManager!
+
+    override func setUp() {
+        super.setUp()
+        
+        offeringsManager = OfferingsManager(deviceCache: mockDeviceCache,
+                                            operationDispatcher: mockOperationDispatcher,
+                                            systemInfo: mockSystemInfo,
+                                            backend: mockBackend,
+                                            offeringsFactory: mockOfferingsFactory,
+                                            productsManager: mockProductsManager)
+    }
+
+}
+
+extension OfferingsManagerTests {
+
+    func testMissingProductDetailsReturnsNil() {
```

Comment:
I usually try to include the name of the method that's under test so it's easier as the class grows to locate the tests for a given method. 

It also makes it much clearer when seeing failed tests results in CI, so you know which method is failing just from the name

i.e.: 

```swift
func testOfferingsForAppUserIDReturnsNilIfMissingProductDetails() {
```

Reply:
I like it! 👌🏼